mlxtend/preprocessing/tests/test_onehot_transactions.py

```diff
@@ -47,13 +47,19 @@ def test_fit():
 
 
 def test_transform():
-
     oht = OnehotTransactions()
     oht.fit(dataset)
     trans = oht.transform(dataset)
     np.testing.assert_array_equal(expect, trans)
 
 
+def test_transform_sparse():
+    oht = OnehotTransactions()
+    oht.fit(dataset)
+    trans = oht.transform(dataset, sparse=True)
+    np.testing.assert_array_equal(expect, trans.todense())
```

Comment:
here, for readability, adding a line 

    assert (isinstance(trans, csr_matrix))

wouldn't hurt :)